src/theme/NavbarItem/NavbarNavLink.tsx

```diff
@@ -38,6 +39,22 @@ export default function NavbarNavLink({
         ),
       };
 
+  if (href?.includes("console.prisma.io")) {
+    return (
+      <BrowserOnly>
```

Comment:
important change 2: when rendering any navbar link that directs to `console.prisma.io` check query params.